src/BlockEntities/NoteEntity.cpp

```diff
@@ -245,8 +245,35 @@ void cNoteEntity::MakeSound(void)
 
 	m_World->BroadcastBlockAction(m_Pos, static_cast<Byte>(instrument), static_cast<Byte>(m_Pitch), E_BLOCK_NOTE_BLOCK);
 
-	// TODO: instead of calculating the power function over and over, make a precalculated table - there's only 24 pitches after all
-	float calcPitch = static_cast<float>(pow(2.0f, static_cast<float>(m_Pitch - 12.0f) / 12.0f));
+	float calcPitch;
+	switch (m_Pitch)
+	{
+		case 0: calcPitch = 0.5f;
+		case 1: calcPitch = 0.5297315471796477f;
+		case 2: calcPitch = 0.5612310241546865f;
```

Comment:
I couldn't bother you to make this into a static function, could I? As this code stands the lack of `break` means execution falls all the way to the bottom and `calcPitch` ends up always with value `2`.

Reply:
You find my lack of breaks disturbing...

I could move this to a static function of course

Reply:
i hope the new commit is in the way you asked for